### PR TITLE
Added symlink to plantuml.jar for compatibility

### PR DESCRIPTION
**Pull request by SommerEngineering**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

I added a symlink for this existing formula for plantuml. Because some tools needed a plantuml.jar file at the path.

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
> Because some tools needed a plantuml.jar file at the path.

Which tools?

File: Formula/plantuml.rb [hunk 1]
```diff
@@ -15,9 +15,12 @@ def install
       #!/bin/bash
       GRAPHVIZ_DOT="#{Formula["graphviz"].opt_bin}/dot" exec java -jar #{prefix}/#{jar} "$@"
     EOS
+    system "ln" " -s" " #{prefix}/#{jar}" " #{prefix}/bin/plantuml.jar"
+    system "chmod" " +x" " #{prefix}/bin/plantuml.jar"
```
Comment (MikeMcQuaid) [inline Formula/plantuml.rb]: We don't support `.jar` files in `bin`.

**Comment by SommerEngineering** [comment]
For example the PlantUMLBuilder: https://github.com/SommerEngineering/PlantUMLBuilder

**Comment by SommerEngineering** [comment]
@MikeMcQuaid: Thanks for your review. Physical, there is no `jar` file at the `bin` directory -- it is just the link. Is this also not allowed? How could I modify the install script in order to add the link to the path? Because if I create the link at the prefix directory, it is no longer available through the path...

**Comment by MikeMcQuaid** [comment]
> Is this also not allowed?

It is not. Instead, consider modifying PlantUMLBuilder to use the correct executable name. Thanks!

**Comment by SommerEngineering** [comment]
Thanks @MikeMcQuaid for the fast feedback. Okay, I have done the change within my tool. I do not know if any other tool need a `plantuml.jar` file. Therefore, I close this request.

**Pull request closed without merging**